OpenSim/Simulation/Test/testForces.cpp

```diff
@@ -901,6 +908,133 @@ void testExpressionBasedBushingForce()
     ASSERT(*copyOfSpring == spring);
 }
 
+void testExpressionBasedBushingForceRotational()
+{
+    using namespace SimTK;
+
+    double mass = 5;
+    double stiffness = 2;
+    double start_theta = Pi/6;
+    double ball_radius = 0.25;
+
+    // Setup OpenSim model
+    Model *osimModel = new Model;
+    osimModel->setName("ExpressionBasedBushingRotationalTest");
```

Comment:
Ah, here's the suffix I was looking for. I suggest being consistent with using either prefixes or suffixes.